### PR TITLE
Countdown to the New Year: 31 Days of Ember Addons - ember-a11y-testing (Dec 10th 2019) 

### LINKED ISSUE #375: Create a blog post for December 10, 2019 about the `ember-a11y-testing` addon
## Goal
Write a short blog post for the December blog series that is similar to the [example post]( https://github.com/ember-learn/ember-blog/blob/be8f29ba0ae8b5367d1758b1a150c1b503812a77/source/2019-12-01-countdown-to-the-new-year-ember-sortable.md). You can find more information on how to write your post in our [Contribution guidelines for DecEmber](https://github.com/ember-learn/ember-blog/blob/master/source/december-2019-blog-series-template.md).

## Criteria

-  [ ] file name should 2019-12-DD-countdown-to-the-new-year-ADDON-NAME.md where DD is replaced with the day and ADDON-NAME is replaced with the hyphenated name of the addon
- [ ] The opening sentence should be updated where COUNT is the sequential ordinal word. 

```
<b>This is the COUNT in our DecEmber series- <i>"Countdown to The New Year: 31 Days of 
Ember Addons"</i>.  We plan to highlight a new addon each day until the new year, and 
we hope you'll join us for the fun!</b>
```

- [ ] The first h2 should be "Day NUM" where NUM is the same as the DD in the date.
- [ ] the link to the addon should point to the page on Ember Observer
- [ ] There should be two sub sections: (both h3)- "What it does" and "Why I like it" 
- [ ] Include whether or not the addon has been updated yet for Ember Octane- if not, invite contributors to help get the addon Octane ready.
- [ ] The closing sentence can be slightly altered but should feel similar: 
"Do you use this addon? Or one like it? We'd love to hear about Ember addons that bring you joy!"

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**
Closes #375

### CODE REVIEW

File: source/2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+alias: 2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md
+responsive: true
+---
+
+**This is the tenth in our DecEmber series- <span style="italics">"Countdown to The New Year: 31 Days of Ember Addons"</span>.  We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 10
+
+Let's talk about [ember-a11y-testing](https://emberobserver.com/addons/ember-a11y-testing).
+
```
Comment (MelSumner) [inline source/2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md]:
```suggestion
READMORE
```

File: source/2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### What It Does
+
+The `ember-a11y-testing` addon is a wrapper around [Deque Labs'](https://github.com/dequelabs)
+[axe-core](https://github.com/dequelabs/axe-core) accessibility(a11y) testing engine that helps you identify accessibility issues in your ember app & provides useful information on how to solve them.
```
Comment (MelSumner) [inline source/2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md]:
```suggestion
[axe-core](https://github.com/dequelabs/axe-core) accessibility(a11y) testing engine that helps you identify accessibility issues in your Ember app & provides useful information on how to solve them.
```

File: source/2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+### Why I Like It
+
+Like many of you out there, I too developed webapps without realizing the impact that accessible apps can have on users. In fact, I 
+started learning about a11y problems through this addon when I added it to a few apps that I was working on when this addon got
+released back in 2015. 
+
+While there's more to building accessible apps that can't be assisted by automated tools like this addon, it does help ensure that
```
Comment (MelSumner) [inline source/2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md]:
```suggestion
While there's more to building accessible apps than automated tools (like this addon) can help with, it does help ensure that 
```

File: source/2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+While there's more to building accessible apps that can't be assisted by automated tools like this addon, it does help ensure that 
+common a11y violations are captured before they can make their way to users.
+
+The best part about this addon is that it lets you conditionally turn off parts or the entire violation report so that you don't have
+to feel overwhelmed when you add this to an existing app. In the teams I've worked with, we usually add an environment variable to control
+that setting so that we can work on addressing those violations whenever we can & run a variation of the nightly test run on CI with the flag enabled to check if the violation count hasn't increased since the previous run.
+
+This addon is Octane-ready. Give it a try today & begin your a11y journey with it.
+
+Do you use this addon? Or one like it? We'd love to hear about Ember addons that bring you joy!
```
Comment (MelSumner) [inline source/2019-12-10-countdown-to-the-new-year-ember-a11y-testing.md:36]:
```suggestion
Do you use this addon? Or one like it? We'd love to hear about Ember addons that bring you joy! 

```

**Review by MelSumner — APPROVED** [review]
Thank you for working on this!! :+1:

**Comment by jenweber** [comment]
This PR also includes code to fix the long-standing bug that had a parse error due to the use of let and arrow functions in a file that does not get babeled.

**Pull request merged**